cost/google/idle_compute_instances/README.md

```diff
@@ -0,0 +1,35 @@
+# Google Idle Compute Instances Policy
+
+## What it does
+
+This Policy Template checks for idle instance in Google Compute Engine and then terminates them upon approval. This is meant to be run as a monthly policy.
```

Comment:
why is it a monthly policy.  that's not even an option.  maybe daily or weekly, but the user should decide how often to run it.